conf_collectd_plugin.py

```diff
@@ -0,0 +1,375 @@
+import sys
+import re
+
+import install_utils as utils
+import config
+
+COLLECTD_HOME = "/etc/collectd"
+COLLECTD_CONF_DIR = COLLECTD_HOME+"/managed_config"
```

Comment:
yeah, it should flag stylistic issues that you should fix.

FYI, this is what pep8 is: https://www.python.org/dev/peps/pep-0008/